tests/integration/components/lt-body-test.js

```diff
@@ -33,73 +34,66 @@ test('row selection - enable or disable', function(assert) {
 
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions canSelect=canSelect}}`);
 
-  let row = this.$('tr:first');
+  let [row] = findAll('tr');
 
-  assert.ok(!row.hasClass('is-selectable'));
-  assert.ok(!row.hasClass('is-selected'));
-  row.click();
-  assert.ok(!row.hasClass('is-selected'));
+  assert.notOk(hasClass(row, 'is-selectable'));
+  assert.notOk(hasClass(row, 'is-selected'));
+  click(row);
+  assert.notOk(hasClass(row, 'is-selected'));
 
   this.set('canSelect', true);
 
-  assert.ok(row.hasClass('is-selectable'));
-  assert.ok(!row.hasClass('is-selected'));
-  row.click();
-  assert.ok(row.hasClass('is-selected'));
+  assert.ok(hasClass(row, 'is-selectable'));
+  assert.notOk(hasClass(row, 'is-selected'));
+  click(row);
+  assert.ok(hasClass(row, 'is-selected'));
 });
 
 test('row selection - ctrl-click to modify selection', function(assert) {
   this.set('table', new Table(Columns, createUsers(5)));
 
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions canSelect=true multiSelect=true}}`);
+  let rows = findAll('tr');
+  let [firstRow, , , middleRow] = rows;
+  let lastRow = rows[rows.length - 1];
```

Comment:
```js
let firstRow = find('tr:first-child');
let middleRow = find('tr:nth-child(4)');
let lastRow = find('tr:last-child');
```

I think that's a bit more expressive. You can tell at a glance what index `middleRow` has.